core/src/main/res/values-de/strings.xml

```diff
@@ -28,6 +29,9 @@
   <!--New episodes fragment-->
   <string name="recently_published_episodes_label">Zuletzt veröffentlicht</string>
   <string name="episode_filter_label">Nur neue Episoden anzeigen</string>
+  <!-- Statistics fragment -->
+  <string name="total_time_listened_to_podcasts">Gesamtzeit gespielter Podcasts:</string>
```

Comment:
This is how translations work. We push the default ``strings.xml`` to transifex. People translate the individual strings; we later pull these. That means that changes to all ``strings.xml`` files except the default one are overwritten once we pull from transifex.